### ISSUE TITLE
Allign visually typeahead component with other components for answers

### ISSUE THREAD

**Issue by blcham** [kbss-cvut/s-forms#257]

**Comment by blcham** [comment]
I suggest to have typeahead value black.

**Comment by LaChope** [comment]
@blcham I am not sure how to handle this... In [intelligent-tree-select](https://github.com/lecbyjak/intelligent-tree-select/blob/1abe02e17b7b6735af2c3d651835be9d985e9810/src/components/IntelligentTreeSelect.js#L381), values that have URL will be treated as links.

Possible solutions:

1. Add an extra flag to parameterize (but I'm scared it would not be great for other app using intelligent-tree)
2. Add a css rule in record-manager-ui to disable links in typeahead (I prefer this solution as this implementation is specific to record-manager)

**Comment by blcham** [comment]
I have tip from ML:
"""
Předejte tree komponentně funkci valueRenderer - viz https://github.com/lecbyjak/intelligent-tree-select/blob/1abe02e17b7b6735af2c3d651835be9d985e9810/src/components/IntelligentTreeSelect.js#L372
"""

**Comment by ledsoft** [comment]
Yeah, the simplest implementation would be to pass the following function as `valueRenderer` prop:
```javascript
const valueRenderer = (children) => <>{children}</>;
```

**Comment by LaChope** [comment]
@blcham I realize now there is no PR to bump s-forms, it was solved here: https://github.com/kbss-cvut/s-forms/pull/263, should I make it?

**Comment by LaChope** [comment]
Closing as tested after merging with https://github.com/kbss-cvut/s-forms/pull/243
